### ISSUE TITLE
Spawn point stuck at 0 and do not spawn

### ISSUE THREAD

**Issue by Jhobean** [Sphereserver/Source-X#424]
Occasionnaly, timer of spawn point are stuck to 0. The NPC is death and the spawn point do not do is job.

Probably cause by sectorSleep. 

It's very problematic because when we entering back the dungeon, there is no NPC and the timer do not reset... The only way is to reset them all by a function or manualy.

I think 2 options:
1- Donc sleep spawn point on sleep
2- When sector go out of sleepmode, wakeup spawnpoint.

Thx for help

![image](https://user-images.githubusercontent.com/51728381/79902544-a8227e80-83df-11ea-8094-8bcef482066f.png)

EDIT:
1- i'm able reproduce on my save but not on vanilla because it,s seem dont have enought spawn point
2- I set sector sleep at 1, remove the npc, set xtimer=120 on the spawn and leave.
3- When i Came back 180 sec later, some or fix at 0 sec

IMPORTANT:  When I set sector sleep to 0, all work well (I dont want set sector sleep to 0)

**Comment by alexdan2** [comment]
I got same problem whit field:

![image](https://user-images.githubusercontent.com/6176986/79976200-81764d80-849c-11ea-85dd-cec1c6cf7acb.png)

After i LOGOUT and wait 10 / 15 mins, and relogin:


![image](https://user-images.githubusercontent.com/6176986/79976243-8f2bd300-849c-11ea-88a7-04523f9290c9.png)

Some fire field stuck to timer 0.

Additional info:
- Using last build
- Using clear save 
- Using Clear Sctipt pack

FIre Field Save Block:
```
[WORLDITEM i_fx_field_fire_ns]
SERIAL=040001fe5
TIMER=0
TYPE=t_spell
LINK=01
ATTR=022
MORE2=01
MOREP=28,10075
P=592,905
```

**Comment by GitMadKing** [comment]
as the pattern seems 1 yes and 1 no
may be related to #412

**Comment by alexdan2** [comment]
@GitMadKing  Yes, i think this issue is conecred by some kind of FORITEM.
If FORITEM if is broken in remove also is broken when sector wakeup and check item.

**Comment by alexdan2** [comment]
I make more test whit SectorSleep = 0 not happen

**Comment by GitMadKing** [comment]
made the test now, 
the issue connected is resolved

**Comment by alexdan2** [comment]
@GitMadKing  On testing :)

**Comment by Jhobean** [comment]
Seem to be more than these timer.. On some player, we have i_memory giving some boost after timer. Sometime, these memory timer stuck at 0.....  Probably the same bug.

**Comment by FiLoUs64** [comment]
Corpse also stock at 0 and dont decay https://cdn.discordapp.com/attachments/476635430821953567/702726172039053382/unknown.png

**Comment by Jhobean** [comment]
I made some test and this build I have no problem:
https://ci.appveyor.com/project/cbnolok/sphereserver-x/builds/31938809/job/vy20xoniwgrcldq4/artifacts

The next one  1.1.0.641, timer is broken.

@alexdan2  can you test with your firewall?

**Comment by alexdan2** [comment]
@Jhobean  i Tested build in your link and work good, timer not stuck to 0 when sector wakeup

**Comment by alexdan2** [comment]
Make a little test on build 3277 whit fire field and no longer stuck to 0 when back to sector. 
Good Work

**Comment by Jhobean** [comment]
Work like a charm! THX! Hope my build pointing helped you.
Good fix on this commit : c97a6bf0f912acd9ad78eabff2e974ecfbad9fa6
